### ISSUE TITLE
'width' is inaccessible due to 'internal' protection level

### ISSUE THREAD

**Issue by YoonjaeYoo** [melvitax/AFViewHelper#1]
I like the concept of `ViewHelper` so I wanted to use but I can't get it compiled.
```
Error:(11, 18) 'width' is inaccessible due to 'internal' protection level
```
My code was:

```swift
import ViewHelper

override func viewDidLoad() {
        let iconView = UIImageView(image: UIImage(named: "Icon"))
        iconView.width(120).height(120).center(to: view)
        view.addSubview(iconView)
}
```

**Comment by YoonjaeYoo** [comment]
FYI, I'm using swift 3

**Comment by melvitax** [comment]
This was fixed in version 4.2.4 released 2 days ago

**Comment by YoonjaeYoo** [comment]
Thanks!
